### DISCUSSION TITLE
sa on this and the next n words?

### DISCUSSION THREAD

**Discussion by proofer** [nvim-mini/mini.nvim#1039] [Q&A] [answered]
For example, I'd like to put the word under the cursor and the following 3 words within a pair of quotation marks. Can I do that?

**Comment by pkazmier** [comment, marked as answer]
Sure, there are several ways you can do that with standard vim motions:

1. Assuming your cursor is on the start of the first word, you can press `sa4e"`, which will add a surrounding (`sa`) around the current position and the end of the next four words (`4e`) using quotation marks (`"`). If you are unsure of how many words, you could do `veeeesa"`, which will give you visual feedback on the words you are going to place quotes around.
2. If your cursor was not at the start of the first word, but in the middle, you could use the "around word" textobject, but you'll have to back off by one character, so you don't quote an empty space at the end. Because you need to adjust the selection, you'll want to use visual mode by pressing `v4awhsa"` or the more interactive version `vawawawawhsa"`. 

I'm sure there are many other ways to do the same, but that should get you pointed in the right direction.

**Reply by acid-bong** [reply]
Or go into visual mode, select the words and then `sa`
